### DISCUSSION TITLE
Add import to Houdini Vertex Animations

### DISCUSSION THREAD

**Discussion by franckaubry** [cables-gl/cables_docs#125] [Feature Requests] [answered]
Please add the ability to import [Houdini Vertex Animations](https://www.sidefx.com/docs/houdini/nodes/out/labs--vertex_animation_textures-3.0.html). This opens up an incredibly bag of tricks for real-time use.

The Houdini tool and some examples are outlined in this video: 
https://www.youtube.com/watch?v=3ep9mkwiOjU

**Comment by franckaubry** [comment]
For examples, the Unity shaders can be found here:
https://github.com/sideeffects/SideFXLabs/tree/Development/unity/shaders

**Reply by cristianvogel** [reply]
super cool idea

**Comment by pandrr** [comment]
as i said, please make an example. i created an exr loading op already waiting for examples

**Comment by franckaubry** [comment]
Several examples from Houdini shared with @pandrr (via Dropbox folder)
Please let me know if you need any other examples or further tests with simple data. I am happy to assist further :)

**Comment by pandrr** [comment, marked as answer]
thanks for the examples!
sadly i did not find good sources of documentation and its very confusing/complex multiple file formats consisting of multiple files, and differences between unity/ue exports.

so i guess ill skip this for now...

also the blender approach from seaflap looks great, so i would recommend trying that.

**Reply by franckaubry** [reply]
Ok. I am reaching out to the Side FX Labs developers to see if they would be willing to answer some questions.

**Reply by franckaubry** [reply]
If I get in touch with the developers from SideFX Labs, are you interested in having a talk with them at all you think?
